WordPress/src/main/res/layout/qrcodeauth_error_view.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/error_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <ImageView
+        android:id="@+id/error_image"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/margin_extra_large"
+        android:layout_marginTop="@dimen/margin_extra_large"
+        android:adjustViewBounds="true"
+        android:contentDescription="@string/qrcode_auth_flow_error_content_description"
+        app:layout_constraintBottom_toTopOf="@+id/error_title"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:src="@drawable/img_illustration_empty_results_216dp" />
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/error_title"
+        style="@style/QRCodeAuth.Title"
+        app:layout_constraintBottom_toTopOf="@+id/error_subtitle"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/error_image"
+        tools:text="No internet connection" />
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/error_subtitle"
+        style="@style/QRCodeAuth.Subtitle"
+        app:layout_constraintBottom_toTopOf="@+id/error_primary_action"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/error_title"
+        tools:text="Need an internet connection to scan a qr code" />
```

Comment:
Np: This hardcoded text can be replaced with `qrcode_auth_flow_error_no_connection_subtitle` in `strings.xml`.

Reply:
I will make this change in the final PR, so I don't have to go back and update the remaining 5 open PRs. Thank you. 🙇